mis_builder/tests/test_data_sources.py

```diff
@@ -85,7 +85,10 @@ def setUp(self):
         # create report
         self.report = self.env['mis.report'].create(dict(
             name='test report',
+            move_lines_source=self.env['ir.model'].search(
+                [('model', '=', 'account.move.line')]).id,
         ))
+        self.report._compute_account_model()
```

Comment:
It should not be necessary to change the test since `move_lines_source` has a default value.